web/src/lib/components/admin-page/settings/library-settings/library-settings.svelte

```diff
@@ -18,7 +18,7 @@
   export let config: SystemConfigDto; // this is the config that is being edited
   export let disabled = false;
 
-  const cronExpressionOptions = [
+  $: cronExpressionOptions = [
```

Comment:
This needs to be reactive because of the translations within?

Reply:
Yes to ensure this gets updated when a new language is selected